modules/reindex/src/test/java/org/elasticsearch/index/reindex/ReindexFromRemoteWithAuthTests.java

```diff
@@ -70,7 +70,6 @@
     @Override
     protected Settings nodeSettings() {
         Settings.Builder settings = Settings.builder().put(super.nodeSettings());
-        // Weird incantation required to test with netty
```

Comment:
This isn't true anymore. There used to be a different, less obvious setting below this line.